### ISSUE TITLE
does not work with threading

### ISSUE THREAD

**Issue by dgpv** [Simplexum/python-bitcointx#29]
```
>>> from bitcointx.core import *
>>> import threading
>>> def f():
...     print(CTransaction())
... 
>>> print(CTransaction())
CBitcoinTransaction(, , 0, 2, CBitcoinTxWitness([]))
>>> t=threading.Thread(target=f)
>>> t.start()
>>> Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib64/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "<stdin>", line 2, in f
  File "/home/tmp/rgb/python-bitcointx/bitcointx/core/__init__.py", line 91, in __call__
    if _thread_local.mutable_context_enabled:
AttributeError: '_thread._local' object has no attribute 'mutable_context_enabled'
```

the thread-local storage needs to be initialized for each thread, and now it is not.

**Comment by dgpv** [comment]
fixed in https://github.com/Simplexum/python-bitcointx/commit/c8ed7358261f3a9dba266acc2eeef1670e97dc21

not closing the issue yet, as the fix is not in the release.

**Comment by dgpv** [comment]
Fixed in v1.0.2
